simulation.py

```diff
@@ -0,0 +1,73 @@
+"""
+A function to simulate alpha
+"""
+
+import copy
+import importlib
+import sys
+from thousandaire.constants import TRADING_CONFIG, DATA_LIST_ALL
+from thousandaire.data_loader import DataLoader
+from thousandaire.simulator import Simulator
+
+DATASET = 'price_dataset'
+
+def initialize():
+    """
+    Initialize data we get.
+    """
+    raw_data = DataLoader(DATA_LIST_ALL).get_all()
+    workdays = raw_data['workdays']
+    invest_data = {index : value
+                   for index, value in raw_data.items() if index != 'workdays'}
+    return_data = {}
+    for key, value in workdays.items():
```

Comment:
We prepare data according to our trading locations. These locations should be in workdays.keys(), but workdays.keys() can cover more locations.
For example, even if we only trade in Taiwan, we can have US workdays in our workdays dataset. In such cases, preparing data for those non-trading locations are just waste of resources.